app/utils/get_filtered_records.py

```diff
@@ -37,6 +37,13 @@ def get_all_records():
 
         # Alias for country name
         fields_list.append(Country.country_name.label("country"))
+        # Aliases for lat lngs
+        fields_list.append(Country.latitude.label("country_latitude"))
+        fields_list.append(Country.longitude.label("country_longitude"))
+        fields_list.append(State.latitude.label("state_latitude"))
+        fields_list.append(State.longitude.label("state_longitude"))
+        fields_list.append(City.latitude.label("city_latitude"))
+        fields_list.append(City.longitude.label("city_longitude"))
```

Comment:
could we just have one long/lat depending on what estimate grade it is to reduce the amount of logic the frontend needs to do? 
(you could also do this in another PR if you want)

Reply:
Will probably address in another PR

Reply:
@joanna-chen created the following ticket for this https://airtable.com/tbli2lWQHAqBa6ZcI/viw8Ro4zYPYcBGXRw/reczv1TbukdkC5fEp?blocks=bipIkdtYHmboUfJut

I think I'm going to return an extra field with the pin lat/lng. Don't wanna remove all the other lat/lngs from the response in case we end up needing them for some reason. Lmk if you'd prefer otherwise.